scripts/src/java/org/oppia/android/scripts/todo/TodoIssueResolvedCheck.kt

```diff
@@ -0,0 +1,105 @@
+package org.oppia.android.scripts.todo
+
+import org.oppia.android.scripts.todo.model.Todo
+import java.io.File
+
+/**
+ * Script for ensuring that all TODOs of the closed issue are resolved.
+ *
+ * Usage:
+ *   bazel run //scripts:todo_issue_resolved_check -- <path_to_directory_root>
+ *   <closed_issue_number> <github_sha>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - closed_issue_number: issue number of the closed issue.
+ * - github_sha: sha of the latest commit on the develop branch.
+ *
+ * Example:
+ *   bazel run //scripts:todo_issue_resolved_check -- $(pwd) 6
+ *   77ff8361b4bde52f695ceb91aa1aab36932a94fe
```

Comment:
```suggestion
 *   bazel run //scripts:todo_issue_resolved_check -- $(pwd)
 *   677ff8361b4bde52f695ceb91aa1aab36932a94fe
```

(prev comment wasn't resolved).

Reply:
Ah! 6 is actuay the example issue number.

Reply:
Now, I have broken this down before 6 so as to avoid confusion. PTAL.